src/linear_algebra/finite_dimensional.lean

```diff
@@ -1125,6 +1125,43 @@ basis_of_linear_independent_of_card_eq_finrank lin_ind (trans s.to_finset_card.s
   ⇑(set_basis_of_linear_independent_of_card_eq_finrank lin_ind card_eq) = coe :=
 basis.coe_mk _ _
 
+variables {ι : Type*} (R : Type*) [comm_ring R] [module R V] [algebra R K] [is_scalar_tower R K V]
```

Comment:
Why is this in this file? It doesn't seem to mention `finite_dimensional` at all!

Reply:
Yes, you are right. The first version was in the finite dimension case but only because that was what I needed and didn't bother to prove the general case. I just pushed a new version with the general case that now lives in `linear_algebra/basis`.